apisix/cli/ops.lua

```diff
@@ -542,6 +543,29 @@ Please modify "admin_key" in conf/config.yaml .
         end
     end
 
+    -- check control port conflict when batch-requests is enabled
+    if enabled_plugins["batch-requests"] and yaml_conf.apisix.enable_control then
```

Comment:
Although this only happens when batch-requests is enabled now, I think we can always check if the port conflicts even batch-requests is not enabled.